src/start_testbuilds.py

```diff
@@ -42,11 +42,13 @@ def run_test_job(commit, codebuild_project, images_str=""):
 
     # For SM tests, if EFA_DEDICATED is True, test job will only launch SM Remote EFA tests,
     # or else will only launch standard/rc tests.
-    # For EC2 tests, if EFA_DEDICATED is True, test job will launch both EFA and non-EFA tests,
-    # or else will only launch non-EFA tests.
-    is_test_efa_dedicated = (
-        config.are_sm_efa_tests_enabled() and "sagemaker" in codebuild_project
-    ) or (config.is_ec2_efa_test_enabled() and "ec2" in codebuild_project)
+    is_test_efa_dedicated = config.are_sm_efa_tests_enabled() and "sagemaker" in codebuild_project
+
+    # For EC2 tests, if HEAVY_INSTANCE_EC2_TESTS_ENABLED is True, the test job will run tests on
```

Comment:
If `HEAVY_INSTANCE_EC2_TESTS_ENABLED` is `True`, the test job will run tests on all instance types, as required by the test function.

If `HEAVY_INSTANCE_EC2_TESTS_ENABLED` is `False`, it will run tests only on small/regular instances.

Clarifying this on comment.

Reply:
Thanks!